homeassistant/components/tradfri/light.py

```diff
@@ -138,7 +134,7 @@ async def async_turn_off(self, **kwargs: Any) -> None:
         if ATTR_TRANSITION in kwargs:
             transition_time = int(kwargs[ATTR_TRANSITION]) * 10
 
-            dimmer_data = {ATTR_DIMMER: 0, ATTR_TRANSITION_TIME: transition_time}
+            dimmer_data = {"dimmer": 0, "transition_time": transition_time}
             await self._api(self._device_control.set_dimmer(**dimmer_data))
```

Comment:
I'd remove the dict unpacking completely.
```suggestion
            await self._api(
                self._device_control.set_dimmer(
                    dimmer=0, transition_time=transition_time
                )
            )
```

Reply:
Yeah, it really doesn't help the reader to do it like it is today.

Reply:
We can do it separately. :+1: